### PR TITLE
1307 parselogs

### PR DESCRIPTION
**Pull request by sanchaymittal**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes: #1307

### CODE REVIEW

File: packages/agents/sdk/src/sdkUtils.ts [hunk 1]
```diff
@@ -50,6 +51,16 @@ export class NxtpSdkUtils {
     return new NxtpSdkUtils(nxtpConfig, logger, chainData);
   }
 
+  public parseConnextTransactionReceipt(transactionReceipt: providers.TransactionReceipt): any {
+    const parsedlogs: any = [];
+    transactionReceipt.logs.forEach((log) => {
+      const l = this.contracts.connext.parseLog(log);
```
Comment (rhlsthrm) [inline packages/agents/sdk/src/sdkUtils.ts:57]: This is gonna throw an error. We should make this function more specific and make the function:
```ts
function getTransferIdFromXCallReceipt(...)
```
Then we know exactly which log in the receipt to parse for.

File: packages/agents/sdk/test/sdkUtils.spec.ts [hunk 2]
```diff
@@ -65,6 +66,14 @@ describe("SdkUtils", () => {
     });
   });
 
+  describe("#parseConnextTransactionReceipt", () => {
+    it("happy: should work", async () => {
+      const res = nxtpUtils.parseConnextTransactionReceipt(mock.ethers.receipt());
```
Comment (rhlsthrm) [inline packages/agents/sdk/test/sdkUtils.spec.ts:71]: Fix this test by using a real xcall receipt.

**Review by rhlsthrm — CHANGES_REQUESTED** [review]

File: packages/agents/sdk/src/sdkUtils.ts (hunk 1 above)
Comment (sanchaymittal) [reply packages/agents/sdk/src/sdkUtils.ts:57]: we can parse any log related to connext interface though, it doesn't have to be xcall right?

File: packages/agents/sdk/test/sdkUtils.spec.ts (hunk 2 above)
Comment (sanchaymittal) [reply packages/agents/sdk/test/sdkUtils.spec.ts:71]: yeah good idea, will add it.

**Pull request merged**